lib/phoenix/live_dashboard/live/page_live.ex

```diff
@@ -106,12 +224,58 @@ defmodule Phoenix.LiveDashboard.PageLive do
   end
 
   @impl true
+  def handle_event("select_node", params, socket) do
```

Comment:
What do you think about us prefixing all of the events here with `pl:select_node` (from page live) or `da:select_node` (dashboard) to avoid conflicts with user events?